src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -205,7 +205,7 @@ private void registerBuiltins() {
         reg(Material.TNT, new BlockTNT());
         reg(Material.DOUBLE_PLANT, new BlockDoublePlant());
         reg(Material.PUMPKIN, new BlockPumpkin());
-        reg(Material.JACK_O_LANTERN, new BlockDirectDrops(Material.JACK_O_LANTERN));
+        reg(Material.JACK_O_LANTERN, new BlockJackOLantern());
```

Comment:
Why create a new class instead of just using the constructor for BlockPumpkinBase?